### ISSUE TITLE
[Bug]: 500 Server Error when unsetting sponsorblock integration that is already unset

### ISSUE THREAD

**Issue by DEJunkins** [tubearchivist/tubearchivist#362]
### Latest and Greatest

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).

### Operating System

Linux/Kubernetes

## Describe the bug
In the channel settings page, if there is a global sponsorblock integration configured but no channel override set, an option to unset sponsorblock integration appears in the drop down to change sponsorblock integrations. Selecting this option and saving channel overwrites results in a 500 Server Error.

## Steps To Reproduce
Enable sponsorblock integration in the global settings
Navigate to a channel page and click the About tab
Verify that Enable SponsorBlock in the about page is False
In the "-- change sponsorblock integrations" dropdown, select "Unset sponsorblock integrations"
Click "Save Channel Overwrites"

## Expected behavior
I would expect the dropdown to either not show an option that is currently set (in this case, unset sponsorblock integration) or for a check to be inserted so that selecting the unset sponsorblock integration and applying when it is already unset won't do anything.

### Relevant log output

```shell
handle post from UCVVcFgllT0oMbEj5OtgzrTw
UCVVcFgllT0oMbEj5OtgzrTw: set overwrites {'download_format': '', 'autodelete_days': None, 'index_playlists': '', 'integrate_sponsorblock': '0'}
UCVVcFgllT0oMbEj5OtgzrTw: get metadata from es
[pid: 40|app: 0|req: 693/693] 10.1.194.0 () {60 vars in 1285 bytes} [Wed Nov  9 14:23:20 2022] POST /channel/UCVVcFgllT0oMbEj5OtgzrTw/about/ => generated 145 bytes in 129 msecs (HTTP/1.1 500) 7 headers in 248 bytes (1 switches on core 0)
[pid: 40|app: 0|req: 694/694] 10.1.194.0 () {52 vars in 972 bytes} [Wed Nov  9 14:23:21 2022] GET /favicon.ico => generated 179 bytes in 3 msecs (HTTP/1.1 404) 7 headers in 228 bytes (1 switches on core 0)
```

**Comment by bbilly1** [comment]
Yeah, I can reproduce that. We'll heave to do some error handling, best to just skip it then. Thanks for reporting.

**Comment by DEJunkins** [comment]
Found the same issue on the global settings page, entering 0 to disable automatic deletion of watched videos when it is already disabled.

**Comment by bbilly1** [comment]
Thanks for reporting, both cases are now fixed in v0.3.0.
